docker/supervisor.conf

```diff
@@ -0,0 +1,21 @@
+[supervisord]
+logfile=/code/logs/supervisord.log
+
+[program:olympia]
+command=python /code/manage.py runserver 0.0.0.0:8000
+directory=/code
```

Comment:
I copied what we had for payments-service https://github.com/mozilla/payments-service/blob/master/docker/supervisor.conf#L5